hack/ci/ci-setup-kubermatic-in-kind.sh

```diff
@@ -172,10 +172,12 @@ retry 5 curl -s --fail http://127.0.0.1:2047/metrics -o /dev/null
 echodate "Cluster exposer is running"
 
 echodate "Setting up iptables rules for to make nodeports available"
-iptables -t nat -A PREROUTING -i eth0 -p tcp -m multiport --dports=30000:33000 -j DNAT --to-destination 172.17.0.2
+KIND_NETWORK_IF=$(ip -br addr | grep -- 'br-' | cut -d' ' -f1)
+
+iptables -t nat -A PREROUTING -i eth0 -p tcp -m multiport --dports=30000:33000 -j DNAT --to-destination 172.18.0.2
 # By default all traffic gets dropped unless specified (tested with docker
 # server 18.09.1)
```

Comment:
is this comment still valid? seems important.

Reply:
I have no idea, actually. I just adjusted what was there until it worked...

Reply:
This variable **KIND_NETWORK_IF**  should return a docker0 interface. 
Is there a specific reason this was changed to match a bridge (I assume)?

Reply:
Also as this is commented, any reason to change the DNAT address?

Reply:
I had to change it, because otherwise there was no connectivity through to the nodeport service. Take this failed job as an example: https://public-prow.loodse.com/view/gs/prow-dev-public-data/pr-logs/pull/kubermatic_kubermatic/5823/pre-kubermatic-e2e-aws-ubuntu-1.19-legacy/1302920728420552704

Reply:
`docker network inspect kind` showed that the kind network is using the same IP range as `ip addr` reported for the bridge interface. See https://github.com/kubernetes-sigs/kind/releases/tag/v0.8.0 for more information.

Reply:
lgtm from my side, I will leave it to @moelsayed